vcf2maf.pl

```diff
@@ -233,10 +228,38 @@ sub GetBiotypePriority {
     'retain-ann=s' => \$retain_ann,
     'min-hom-vaf=s' => \$min_hom_vaf,
     'remap-chain=s' => \$remap_chain,
-    'max-subpop-af=f' => \$max_subpop_af
+    'max-subpop-af=f' => \$max_subpop_af,
+    'samtools-exec=s' => \$samtools,
+    'tabix-exec=s' => \$tabix,
+    'liftover-exec=s' => \$liftover,
```

Comment:
Note that the trailing comma is intentional and allowed in Perl.  If future command line options are added at the end, the previous last line won't need to show a change.